assets/js/hooks/message_settings.js

```diff
@@ -1,53 +1,118 @@
-import { extractActionAndMessageIdFromDomElementId } from "../utils"
+import { extractActionAndMessageIdFromDomElementId } from "../utils";
 
 function triggerMessageDeletionEvent(event, component) {
-    if (event.relatedTarget != undefined) {
-        elementType = event.relatedTarget.localName 
-        elementAction = extractActionAndMessageIdFromDomElementId(event.relatedTarget)[0]
-        if (elementType != undefined && elementType == "button" && elementAction == "delete") {
-            component.pushEvent("delete_message", {"id": extractActionAndMessageIdFromDomElementId(event.srcElement)[1]}) 
+  if (event.relatedTarget != undefined) {
```

Comment:
If a condition has to be met if something is not nothing, it means that the condition will be true if that something exist. So. `if(event.relatedTarget)` or `if(!!event.relatedTarget)` should have the same effect.